fastai/sgdr.py

```diff
@@ -514,8 +514,6 @@ def phase_begin(self, layer_opt, nb_batches):
         self.layer_opt.set_mom(start_mom)
         if self.beta is not None: self.layer_opt.set_beta(self.beta)
         if self.wds is not None:
-            if not isinstance(self.wds, Iterable): self.wds=[self.wds]
-            if len(self.wds)==1: self.wds=self.wds*len(self.layer_opt.layer_groups) 
```

Comment:
Is this not missing the replacement line? `self.wds = listify(self.wds, self.layer_opt.layer_groups)`

Reply:
There are listified inside layer_opt with the next lines so I didn't bother doing it here.